channels/ON-CHAIN.md

```diff
@@ -252,26 +253,32 @@ Serialization defined [here](../serializations.md#channel-close-solo-transaction
 - `nonce`: taken from the `from`'s account
 - `fee`
 
-Proof of inclusion represents the internal channel. At the
-bare minimum it has to include all accounts and their balances but can also
-include contracts and contract calls. It must provide enough information to
+Proof of inclusion represents the internal channel' s state. At the
```

Comment:
channel's ?

Reply:
Yes - the party that forces progress provides a `proof of inclusion` that represents the state of the channel so this internal channel state can be updated on-chain.
A simplified example: Alice and Bob have a channel and there is some `Contract1` being present in their off-chain channel state. Alice wants to execute a call to `Contract1` but Bob either is missing or just refuses to play along. In this case Alice unilaterally forces this contract execution on-chain.

In order to do this - Alice brings the off-chain state on-chain: the acounts and contracts. The `Contract1` is being executed on-chain, the off-chain is updated and a new channel's state is produced. Alice and Bob can continue from there.

So Alice provides _some_ state that she has to prove to be a valid channel's state. This is the same `proof of inclusion` being refered in your comment. Alice has also to provide a *co-signed* off-chain state (the `payload`) that has the same `state_hash` as it is the `root_hash` of the provided `proof of inclusion`. This is how Alice proves that this is a valid channel's state both parties had agreed upon (at some point of time at least). This could as well be some old state and in this case Bob has to dispute it.

Having provided a channel's state (being the `proof of inclusion` with a correct root hash) - Alice can unilaterally call a contract that had been created off-chain on-chain. This way Alice uses the blockchain as an arbiter in her case with Bob. We use the state provided by Alice to produce the next channel's state.

Reply:
Got it, thanks @velzevur, there is a mistype space in `channel' s`.

Reply:
Got it, really thanks @velzevur, there is a mistype space in `channel' s` ; ).

Reply:
Ah, thank you